packages/cli/src/commands/index.ts

```diff
@@ -222,6 +222,14 @@ export function registerCommands(program: CommanderStatic) {
       '--pattern <glob>',
       'Override glob for matching packages to upgrade',
     )
+    .option(
+      '--release-line <main|next>',
+      'Bump to the latest version of a specific release line',
+    )
+    .option(
+      '--backstage-release <version>',
```

Comment:
Probably worth bike-shedding these two a bit. One possibility here since `main` is the default is to make the release line flag simply `--next`, neater but not very future proof. I'm also not 100% sure on the "release line" wording over all, are there some other options there that we haven't considered? ^_>

Regarding the last one, is the `backstage-` prefix really worth it? :grin: We're solidly in a Backstage context after all. Option here is ofc to go with just `--release <version>`. It is kinda nice to avoid the risk of confusion though.

Another slightly more crazy idea is `--manifest https://versions.backstage.io/v1/versions/releasts/0.1.0/manifest.json`